ArcGISRuntimeSDKQt_CppSamples/Scenes/ExtrudeGraphics/ExtrudeGraphics.cpp

```diff
@@ -107,8 +107,9 @@ void ExtrudeGraphics::componentComplete()
   for (const Point& point : pointsList)
   {
     // create a random z value
-    int randNum = rand() % 6 + 1;
-    double z = m_maxZ * randNum;
+    const int randNum = rand() % 6 + 1; // security.insecureAPI.rand: Warn on uses of inferior random number generating functions (only if arc4random function is available): drand48, erand48, jrand48, lcong48, lrand48, mrand48, nrand48, random, rand_r.
```

Comment:
// security.insecureAPI.rand: Warn on uses of inferior random number generating functions (only if arc4random function is available): drand48, erand48, jrand48, lcong48, lrand48, mrand48, nrand48, random, rand_r.

Reply:
I have no recommendations here. Sounds like we should fix the warnings, but I will defer to the team on recommendations.

Reply:
I keep like that. I guess we can ignore this warning.